src/main/resources/project/jython/preamble.jython

```diff
@@ -41,3 +42,22 @@ def parseOptions(options):
         arguments = nextArgument.split('=')
         optionsDict[arguments[0]]=arguments[1]
     return optionsDict
+
+# checks if application exists:
+def checkIfApplicationExists(appName, appVersion):
+    print 'Checking AppName: %s with Version: %s for existance' % (appName, appVersion)
+    for deployment in cmo.getAppDeployments():
+        tempName = deployment.getName()
+        name = ''
+        version = ''
+        if re.match('#', tempName):
```

Comment:
If an application has a plan version also, there will be two sharps in the application name, like HelloWorld#beta#1